web/src/engine/websites/Akuma.ts

```diff
@@ -0,0 +1,47 @@
+import { Tags } from '../Tags';
+import icon from './Akuma.webp';
+import { Chapter, DecoratableMangaScraper, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+const pageScript = `
+    new Promise( (resolve, reject) => {
+        try {
+            const url = window.location.href.replace(/\\/1$/, '');
+            const token = document.querySelector('meta[name="csrf-token"]').content.trim();
+            fetch(url, {
+                credentials: 'include',
+                headers: {
+                    'X-CSRF-TOKEN': token,
+                    'X-Requested-With': 'XMLHttpRequest',
+
+                },
+                 origin: window.location.origin,
+                 referrer: window.location.href,
+                 method: 'POST',
+            }).then(response => response.json()
+                .then(data => resolve(data.map(image => img_prt + '/' + image))));
+
+        } catch (error) {
+            reject(error);
+        }
+    });
+ `;
+
+@Common.MangaCSS(/^{origin}\/g\/[^/]+$/, 'h1.entry-title')
+@Common.MangasNotSupported()
+@Common.PagesSinglePageJS(pageScript)
```

Comment:
Maybe extracting the token in `Initialize()` and store it as field, so the JSON API can be accessed directly when requesting pages.

Reply:
I'll still need to get the javascript variable of the CDN in chapter page so i dont think its worth.